Sources/SwiftGodot/SwiftGodot.docc/Tutorials/Your First Extension.tutorial

```diff
@@ -144,6 +144,8 @@
             
         @Steps {
             @Step {
+                Download the provided Example project **SimpleRunner-Starter.zip** at the top of the page.
+
```

Comment:
Most tutorials I've seen that are written with DocC don't explicitly tell the developer to download the package, since it's clearly visible. I think this is a little redundant.

Reply:
I created a fresh Godot project only to later realize that I should have used the provided example.

Reply:
I feel that there’s already enough context provided in the tutorial to indicate that creating an entirely new project is _not_ the idea:

- First, the abstract of the tutorial states that one will be building off the Simple Runner project, indicating that there is already a starter project to work off of. The abstract is also the first visible thing, since it outlines tutorial length, starter files, etc., and should _not_ be ignored or glossed over.
- Second, the tutorial never calls for making an entirely new Godot project, since the starter already has most of the inner workings together.
- Finally, I feel that adding an instruction to download the starter code when the abstract is already visible may come off as derogatory or demeaning to the developer, belittling them in a way.